kythe/proto/generate_protobufs.py

```diff
@@ -35,47 +39,64 @@
 import stat
 import sys
 
+parser = argparse.ArgumentParser(description='Generate Go or Rust protobufs')
+parser.add_argument('--rust', dest='lang', action='store_const', const='rust',
+                    default='go', help='generate Rust instead of Go')
```

Comment:
Any particular reason for this not to be a list such that it generates both Rust and Go?

The odds are good that any change to the protos will necessitate a change to both Rust and Go generated protos so being able to do so in a single command would be nice.

`parser.add_argument('--languages', default = ['go', 'rust'], etc.)` ?

Reply:
I just dropped the option entirely.